assets/js/googlesitekit/datastore/user/user-info.js

```diff
@@ -133,16 +134,20 @@ export const actions = {
 	 * Stores the user input state in the datastore.
 	 *
 	 * @since 1.20.0
+	 * @since n.e.x.t Updated the state object to be a simple boolean.
```

Comment:
This is a completely new action, so preserving the since history here would be inaccurate. This should simply be `@since n.e.x.t` just as any other new action/selector.